zombienet/README.md

```diff
@@ -0,0 +1,52 @@
+**Zombienet for macos**
+
+Download manualy from [zombienet](https://github.com/paritytech/zombienet/releases)
+
+or using *wget* command:
+
+`wget https://github.com/paritytech/zombienet/releases/download/v1.3.30/zombienet-macos`
+
+- Give permission to execute *zombienet-macos* file
+
+`chmod +x zombienet-macos`
+
+- Check `--help`
+
+`./zombienet-macos --help`
+
+**Build Polkadot**
+
+- clone polkadot
+
+```
+git clone git@github.com:paritytech/polkadot
+cd polkadot
+```
+
+- build polkadot with *testnet* profile or production
+
+`cargo build --profile testnet -p test-parachain-adder-collator` or `cargo build --release`
```

Comment:
What's the `test-parachain-adder-collator` package? Is it just an example? Why does the testnet profile specify that package and the release one does not?

Reply:
You right. there is no needs to add this package. just example. 
Removed.